timesync

```diff
@@ -99,7 +103,27 @@ def parse_all_args():
     return parser.parse_args()
 
 
+def read_config():
+    """Returns a dictionary representing the contents of the json file
+    $HOME/timesyncconf.json or a set of sensible defaults if that file does
+    not exist.
+    """
+    sensible_defaults = {
+        'url': 'http://localhost:3000',
+        'editor': 'vi',
```

Comment:
This is in preparation for the next feature.
My idea was that this would be a git post-commit hook which would have a UX similar to COMMIT_EDITMSG. When  you write your commit message vim opens and you edit a file called COMMIT_EDITMSG. Once you save the file git checks that there are lines which aren't comments and makes that the commit message. After that, the post commit hook triggers, and vim opens again so you can edit your check in.
Does this make sense?